pkg/service/presentation/storage/bolt.go

```diff
@@ -70,3 +71,40 @@ func (b BoltPresentationStorage) DeletePresentation(id string) error {
 	}
 	return nil
 }
+
+func (b BoltPresentationStorage) StoreSubmission(submission StoredSubmission) error {
+	id := submission.Submission.ID
+	if id == "" {
+		err := errors.New("could not store submission definition without an ID")
+		logrus.WithError(err).Error()
+		return err
+	}
+	jsonBytes, err := json.Marshal(submission)
+	if err != nil {
+		errMsg := fmt.Sprintf("could not store submission definition: %s", id)
+		logrus.WithError(err).Error(errMsg)
+		return errors.Wrapf(err, errMsg)
```

Comment:
nit: could use the `util.Logging...` whatever as it combines the error creation, logging, and error return here and the rest of this file. but NBD

Reply:
Done.